### DISCUSSION TITLE
A value is required for '-b <BOOTNODES>' but none was supplied

### DISCUSSION THREAD

**Discussion by Tejaswini-Pardeshi** [NillionNetwork/discussions#73] [General]
Hi,
When i am setting up the nillion python starter, getting the error while storing the compiled nada.bin file into the nillion network.
Setup Information - 
1. nilup version - 0.2.1
2. python client version - 0.2.1

![Screenshot from 2024-06-27 16-16-24](https://github.com/NillionNetwork/discussions/assets/110804199/b76a81ab-e62e-4b77-b836-ee2bcf83a452)
![Screenshot from 2024-06-27 16-16-03](https://github.com/NillionNetwork/discussions/assets/110804199/56198ef9-80ef-40f5-a701-25e89ba4e33b)

Cluster information not fetched correctly...Can you please help me with this...

**Comment by Davetbutler** [comment]
Hey. To help me help you here, which version of the SDK did you install? When you ran `nilup install <version>` ? I think there is a simple mismatch in the quickstart version and the SDK version... :)

**Reply by Tejaswini-Pardeshi** [reply]
I have installed the 0.2.1 version of the SDK...

**Reply by wwwehr** [reply]
when you run the `nillion-devnet` you will get an autogenerated .env file listed in the output. you shall use this file, or the contents, to configure your python application.

**Reply by wwwehr** [reply]
default is `~/.config/nillion/nillion-devnet.env` - see this screenshot

![image](https://github.com/NillionNetwork/discussions/assets/33910651/9d97d8af-d35c-49dc-a553-f396be74908d)

**Reply by Tejaswini-Pardeshi** [reply]
I am referring this python client starter...https://github.com/NillionNetwork/nillion-python-starter/tree/payments-flow-update

![Untitled design](https://github.com/NillionNetwork/discussions/assets/110804199/d84ff7ba-29b4-4fce-89c1-868260b48f90)

Can you please share me the js client repo link you are referring to....

**Reply by wwwehr** [reply]
that is the one. make sure you are following the instructions on the nilup changes to activate the version called `latest-experimental`. I think that's the thing you're missing!!! let me know.
